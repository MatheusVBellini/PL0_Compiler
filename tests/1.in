VAR x;
BEGIN
  x := 5;
  WHILE x < 0
END.
