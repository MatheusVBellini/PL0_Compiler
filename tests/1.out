3:5 error: Invalid symbol
3:5 error: Missing equal `=` symbol after this token

Compilation failed, 2 errors
